libs/sdk-ui-pivot/src/impl/resizing/columnSizing.ts

```diff
@@ -870,19 +872,27 @@ function getTableFonts(containerRef: HTMLDivElement): {
 /**
  * Ag-Grid API set desired column sizes (it *mutates* pivot table columns data).
  */
-export function autoresizeAllColumns(columnApi: ColumnApi | null, autoResizedColumns: IResizedColumns): void {
+export async function autoresizeAllColumns(columnApi: ColumnApi | null, autoResizedColumns: IResizedColumns) {
     if (columnApi) {
         const columns = columnApi.getPrimaryColumns();
 
-        columns.forEach((column: Column) => {
-            const columnDef = column.getColDef();
-            const colId = agColId(columnDef);
-            const autoResizedColumn = autoResizedColumns[colId];
-
-            if (colId && autoResizedColumn && autoResizedColumn.width) {
-                columnApi.setColumnWidth(colId, autoResizedColumn.width);
-            }
-        });
+        const chunks = chunk(columns, COLUMN_RESIZE_CHUNK_SIZE);
```

Comment:
Maybe we should add just small comment about the purpose of this code?

Reply:
Added the comment